syntax = "proto3";

package helloworld;

// 定义请求
message HelloRequest {
  string name = 1;
  string id = 2;
}

// 定义应答
message HelloReply {
  string message = 1;
  string id = 2;
}

// 定义服务
service Greeter {
  rpc SayHello(HelloRequest) returns (HelloReply);
}